tracker/tracker_debugger_api.yaml

```diff
@@ -286,10 +287,15 @@ components:
           type: integer
           example: 8
         event_url:
+          format: url
+          description: A URL on which event occurred, human-readable
```

Comment:
Looks good, but `human-readable` is very enigmatic way to distinguish transformation to UTF8 form od URI. We can leave it, but maybe we should define in glossary what it means and link there.

Reply:
I've added an entry to our private glossary:
https://piwikpro.atlassian.net/wiki/spaces/PPTP/pages/85731116046/Glossary